elasticsearch/manifests/configmap.yaml

```diff
@@ -31,7 +31,8 @@ data:
   post-start-hook.sh: |-
     #!/bin/bash
     set -e
-    apk --no-cache add jq
+    yum install -y epel-release
+    yum install -y jq
```

Comment:
Can we remove these?

Reply:
Otherwise need need to upstream this.  Also do we have the RBAC roles for elastic?  How long do we have for the pod shutdown time?